tools/reflection_generator/java_method.py

```diff
@@ -269,6 +270,12 @@ def ParseMethodAnnotation(self, annotation):
       post_wrapline = self.FormatWrapperLine(match.group('post_wrapline'))
       self._method_annotations[self.ANNOTATION_POST_WRAPLINE] = post_wrapline
 
+    post_bridline_re = re.compile('postBridgeLines\s*=\s*\{\s*('
```

Comment:
`brid` or `bird`? Anyway, I don't know why to prefer `post_brid_line` to `post_bridge_line`